_sass/custom/custom.scss

```diff
@@ -0,0 +1,35 @@
+kbd {
+    // TODO: use just the docs variables?
+    --kbd-color-background: #e6e1e8;
+    --kbd-color-border: #959396;
+    --kbd-color-text: #27262b;
+
+    // Match the page defaults in light mode but
+    // switch to a light-ish grey in dark mode
```

Comment:
Not sure what this comment is referring to as there's no `prefers-color-scheme` stuff around.
Does the F3D website have a light mode or is it dark only?

Reply:
I copy-pasted a random code from internet and realized it was not needed but forgot to remove the comment ^^